torch/_logging/_registrations.py

```diff
@@ -0,0 +1,26 @@
+from ._internal import register_artifact, register_log
+
+
+TORCHDYNAMO_LOG_NAME = "torch._dynamo"
+TORCHINDUCTOR_LOG_NAME = "torch._inductor"
+AOT_AUTOGRAD_LOG_NAME = "torch._functorch.aot_autograd"
+
+# (optional) shorthand names used in the logging env var and user API
+TORCHDYNAMO_NAME = "dynamo"
+TORCHINDUCTOR_NAME = "inductor"
+AOT_AUTOGRAD_NAME = "aot"
```

Comment:
Wow, this file is so short and sweet now, awesome! Nit: the constants like TORCHINDUCTOR_NAME are only used in one place now, might as well inline them.

Reply:
Yeah this got really simplified, and with the lazy artifact init, I really wanted to not need to register the log where the artifact gets logged, and it worked!